<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>321</width>
      <height>192</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">gridline-color: rgb(143, 240, 164);</string>
    </property>
    <column>
     <property name="text">
      <string>nome</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>valor</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>data</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton_visual">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Visualizar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_voltar">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>260</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Voltar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>425</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
